### ISSUE TITLE
Add an input gesture to change the mouse text unit resolution value

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#9056]
### Is your feature request related to a problem? Please describe.
In some cases, I use the mouse tracking feature of NVDA to read text that is under the mouse cursor. I've set the mouse "text unit resolution" value to "line" and for most cases until recently, it was OK. But now that NVDA's mouse tracking works more as it should in regards to text unit resolution in places such as Microsoft Edge, I've started thinking that in some cases reading by line is OK, but in some other cases, reading by paragraph is better. So, a way to switch quickly between the text unit resolution modes (character, word, line, paragraph) would be welcome. I've looked and there is no NVDA input gesture for that at the moment. The only way to change the mouse text unit resolution is to go into the NVDA's settings, in the "Mouse" category and change it from there.

### Describe the solution you'd like
So, I think that there should be a way to assign an NVDA input gesture to quickly change the value for the mouse text unit resolution.

### Describe alternatives you've considered
Changing the value for the mouse text unit resolution each time I want to change it via NVDA's settings panel takes too many steps. Changing it via configuration profiles also seams a bit too complicated, considering in most cases I want to change it quickly and temporarily for a specific case.
